.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you update the `.gitignore` with `.vscode/*` as well?

Reply:
Absolutely!